runtime/src/main.rs

```diff
@@ -46,6 +46,7 @@ pub extern "C" fn entry_point() -> ! {
                 );
             }
         };
+        caliptra_common::wdt::stop_wdt(&mut drivers.soc_ifc);
```

Comment:
If you handle the wdt in wait_for_cmd, you can probably remove this line.

Reply:
When/if wfi is implemented , it will make sense.